Ruby-Project/person_class.rb

```diff
@@ -23,4 +25,31 @@ def of_age?
 
     end
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] This should not be a private method. Kindly remove it from under the `private` keyword.

- [ ] Kindly fix these broken links in your readme headers. You can remove those sections if you have no sections intended for them.

  ![image](https://github.com/OnlyUnknown/OOP-school-library/assets/81468675/79c31e4d-7283-4abe-8498-07b523fbb26e)